testonly/hammer/hammer.go

```diff
@@ -165,8 +166,26 @@ func (c MapConfig) String() string {
 		c.MapID, c.EPBias, c.Operations, c.EmitInterval, c.RetryErrors)
 }
 
+// SetULimit calls `ulimit -n` to set the maximum number of open files.
+// See https://wilsonmar.github.io/maximum-limits/
+func SetULimit(uLimit uint64) error {
```

Comment:
Rename to `SetFDLimit(fdLimit uint64)`? 
(`SetULimit` seems like it should allow the caller to change any of the posix user limits)